### PR TITLE
[Merged by Bors] - style: a linter for colons

### PR DESCRIPTION
**Pull request by BoltonBailey**
A linter that throws on seeing a colon at the start of a line, according to the [style guideline](https://leanprover-community.github.io/contribute/style.html#structuring-definitions-and-theorems) that says these operators should go before linebreaks. 

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by BoltonBailey** [comment]
I saw someone touch this file so I thought I might as well un-draft this PR. I guess we want this whole script to be translated from Python to Lean, but this is a check I think I would find helpful.

File: Mathlib/CategoryTheory/Limits/Shapes/FiniteLimits.lean [hunk 1]
```diff
@@ -114,9 +114,9 @@ theorem hasFiniteColimits_of_hasFiniteColimits_of_size
     HasFiniteColimits C where
   out := fun J hJ hhJ => by
     haveI := h (ULiftHom.{w} (ULift.{w} J)) <| @CategoryTheory.finCategoryUlift J hJ hhJ
-    have l :
-      @Equivalence J (ULiftHom (ULift J)) hJ (@ULiftHom.category (ULift J) (@uliftCategory J hJ))
-      := @ULiftHomULiftCategory.equiv J hJ
+    have l :  @Equivalence J (ULiftHom (ULift J)) hJ
```
Comment (jcommelin) [inline Mathlib/CategoryTheory/Limits/Shapes/FiniteLimits.lean:117]:
```suggestion
    have l : @Equivalence J (ULiftHom (ULift J)) hJ
```

**Review by jcommelin — APPROVED** [review]
Thanks so much!

bors d+

**Pull request closed without merging**